thirdeye-ui/src/app/components/alert-view/enumeration-items-table/enumeration-item-row/enumeration-item-row.component.tsx

```diff
@@ -138,12 +197,28 @@ export const EnumerationItemRow: FunctionComponent<EnumerationItemRowProps> = ({
                                     )
                                 }
                             >
-                                {!isExpanded && (
-                                    <span>{t("label.view-details")}</span>
-                                )}
-                                {isExpanded && (
-                                    <span>{t("label.hide-details")}</span>
-                                )}
+                                <Box
+                                    alignItems="center"
+                                    component="span"
+                                    display="flex"
+                                >
+                                    {!isExpanded && (
+                                        <>
+                                            <span>
+                                                {t("label.view-details")}
+                                            </span>
+                                            <ExpandMoreIcon />
+                                        </>
+                                    )}
+                                    {isExpanded && (
+                                        <>
+                                            <span>
+                                                {t("label.hide-details")}
+                                            </span>
+                                            <ExpandLessIcon />
+                                        </>
+                                    )}
```

Comment:
Can be simplified to:

```
<span>
    {t("label.view-details")}
</span>
{!isExpanded && (
    <ExpandMoreIcon />
)}
{isExpanded && (
    <ExpandLessIcon />
)}
```

Reply:
The labels are different, `view-details` and `hide-details`, so need to have them in the condition

Reply:
Ohhh I see